pkg/api/user.go

```diff
@@ -250,6 +250,11 @@ func ChangeUserPassword(c *models.ReqContext, cmd models.ChangeUserPasswordComma
 	return Success("User password changed")
 }
 
+// GET /.well-known/change-password
```

Comment:
Declaration doc comments should be sentences describing the thing in question:
```suggestion
// RedirectToChangePassword handles GET /.well-known/change-password.
```

Reply:
Was following the pattern established by other functions in this file

Reply:
Yeah, there's a lot of technical debt in the Grafana code base :) When we get proper doc comment linting enabled (it's a bit tricky) it'll catch this.